examples/js/controllers.js

```diff
@@ -4058,12 +4058,12 @@ var app = angular.module('webapp');
             });
         }, 4000);
         angular.extend($scope, {
-            markersWatchOptions: {
-                doWatch: false,
-                isDeep: false,
-                individual: {
-                    doWatch: false,
-                    isDeep: false
+            watchOptions: {
+                markers: {
```

Comment:
If watches were off by default, then I would support `undefined` or `null` as a way to turn off watching, however, watches are `watchDeep` by default as this was how angular-leaflet-directive was originally designed (paths have a bit different defaults). I would argue the absence of a property should probably yield the default behaviour. I would support replacing 'none' with `null` (explicitly defined) if so desired. Supporting both is more hassle than it is worth as some conditionals check whether a watch is set simply by checking if `type != 'none'` (supporting both would require longer conditionals).

```
watchOptions = {
   markers: {

   }
}
```